endpoints/contractors.md

```diff
@@ -67,18 +68,102 @@ curl --location --request POST http://api.gearbox.com.au/public/v1/contractors \
 --header "Content-Type: application/json" \
 --header "Authorization: Bearer $ACCESS_TOKEN" \
 --data '{
-      "company_name":"Contractors R Us",
-      "terminated":false,
-      "phone":"123456",
-      "mobile":"0411111111",
-      "address1":"123 Fake Street"
-      "city":"Adelaide",
-      "state":"SA",
-      "postcode":"5005",
-      "position":"CEO",
-      "abn":"1938431907",
-      "email":"contractors@contractorsrus.com.au",
-      "vehicle_group":"Adelaide"
+      "company_name": "Contractors R Us",
+      "terminated" :false,
+      "phone": "123456",
+      "mobile": "0411111111",
+      "address1": "123 Fake Street"
+      "city": "Adelaide",
+      "state": "SA",
+      "postcode": "5005",
+      "position": "CEO",
+      "abn": "1938431907",
+      "email": "contractors@contractorsrus.com.au",
+      "vehicle_group": "Adelaide"
```

Comment:
@ZaidYounan should this field and `vehicle_sub_group` be changed on the Contractor API to match the other endpoints (e.g. `group` / `sub_group`)?

Reply:
It would make sense to me since they both refer to the same field on our end. By originally naming them `group, subgroup` I was just following the specification outlined in the original issues (which I assumed perhaps came directly from the customer).